samples/bluetooth/peripheral_uart/minimal.conf

```diff
@@ -0,0 +1,107 @@
+#
```

Comment:
Please change this file into a full Kconfig file and name it `prj_minimal.conf`, so that we can utilize this principle:
https://docs.zephyrproject.org/latest/guides/kconfig/setting.html#the-initial-configuration
> Otherwise if CONF_FILE is set, and a single configuration file of the form prj_<build>.conf is used, then if file boards/<BOARD>_<build>.conf exists in same folder as file prj_<build>.conf, the result of merging prj_<build>.conf and boards/<BOARD>_<build>.conf is used.

that will allow to add board specific fragments, as example: `boards/nrf52840dk_nrf52840_minimal.conf`

Also, it will allow to use this principle for multi image build, as example `nrf5340dk_nrf5340` where you might want a specific minimal config for the netcore, as example:
`<app>/child_image/hci_rpmsg_minimal.conf`, as introduced by this PR: #3247